### DISCUSSION TITLE
Disable viewer

### DISCUSSION THREAD

**Discussion by sadsasuke** [stella-cv/stella_vslam#391] [Q&A] [answered]
Hi,
How to disable pangolin viewer so the whole algorithm runs and outputs trajectory files without opening any viewer!
Thanks

**Comment by ymd-stella** [comment, marked as answer]
See https://github.com/stella-cv/stella_vslam/blob/14b1866602150706986351f05338deaf34c5d4cf/CMakeLists.txt#L28
